integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/AbstractBridge.java

```diff
@@ -0,0 +1,68 @@
+package com.redhat.service.bridge.integration.tests.common;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.keycloak.representations.AccessTokenResponse;
+
+import com.redhat.service.bridge.test.resource.KeycloakResource;
+
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+public abstract class AbstractBridge {
+
+    private static final String USER_NAME = "kermit";
+    private static final String PASSWORD = "thefrog";
```

Comment:
We can test in environments using different credentials and client config, the parameters need to be configurable, ideally in Maven so they can be specified as command parameters.